File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/helperFunctions/Logger.java
package com.atomykcoder.atomykplay.helperFunctions;

import android.util.Log;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class Logger {
    /**
     * Tag is "DebugTag"
     * @param s
     */
    public static void normalLog(String s) {
        Log.d("DebugTag", s);
    }
}

File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/helperFunctions/Logger.kt
package com.atomykcoder.atomykplay.helperFunctions

import android.util.Log

object Logger {
    /**
     * Tag is "DebugTag"
     *
     * @param s
     */
    fun normalLog(s: String?) {
        Log.d("DebugTag", s!!)
    }
}
